react/features/mobile/navigation/actions.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import { NAVIGATE_TO_ROOM, NAVIGATE_TO_WELCOME } from './actionTypes';
```

Comment:
I created two actions, but maybe there is a way I can conditionally navigate and merge them into one. 
I tried, but didn't find a solution at this moment.